boards/arc/nsim/doc/index.rst

```diff
@@ -7,20 +7,22 @@ Overview
 ********
 
 This board configuration will use `Designware ARC nSIM`_ to emulate a virtual
-ARC EM based board including the following features:
+ARC EM or ARC HS based board including the following features:
 
-* ARC EM processor
+* ARC EM or ARC HS processor
 * ARC internal timer
 * a virtual output only console (uart-nsim)
 
-There are two sub configurations in board:
+There are three supported board sub-configurations:
 
-* nsim_em which includes normal em features and ARC MPUv2
-* nsim_sem which includes secure em features and ARC MPUv3
+* ``nsim_em`` which includes normal ARC EM features and ARC MPUv2
+* ``nsim_sem`` which includes secure ARC EM features and ARC MPUv3
+* ``nsim_hs`` which includes normal ARC HS features
```

Comment:
what does 'normal' mean in HS context? Guess MMU and MPU are not (yet) supported?
Maybe call it 'base' instead of normal.

Reply:
Correct! It was just a copy-paste of EM's description.

So what if I say:
```
nsim_hs which includes base ARC HS features, i.e. without PMU & MMU
```
?